docs/tutorials/multimodal/efficient_finetuning_basic.md

```diff
@@ -0,0 +1,132 @@
+                          sep="\t",
+                          header=None,
+                          names=["label", "text"]) \
+               .sample(200, random_state=123).reset_index(drop=True)
+test_de_df = pd.read_csv("amazon_review_sentiment_cross_lingual/de_test.tsv",
+                          sep="\t", header=None, names=["label", "text"]) \
+               .sample(200, random_state=123).reset_index(drop=True)
+
+test_jp_df = pd.read_csv('amazon_review_sentiment_cross_lingual/jp_test.tsv',
+                          sep='\t', header=None, names=['label', 'text']) \
+               .sample(200, random_state=123).reset_index(drop=True)
+train_en_df.head(5)
+```
+
+
+```{.python .input}
+test_jp_df.head(5)
+```
+
+## Finetuning with IA3 + BitFit
+
+In AutoMM, to enable efficient finetuning, just specify the `optimization.efficient_finetune` to be `"ia3_bias"`.
+
+```{.python .input}
+from autogluon.multimodal import MultiModalPredictor
+
+predictor = MultiModalPredictor(label="label",
+                                path="multilingual_ia3")
+predictor.fit(train_en_df,
+              presets="multilingual",
+              hyperparameters={
+                  "optimization.efficient_finetune": "ia3_bias",
+                  "optimization.lr_decay": 0.9,
+                  "optimization.learning_rate": 3e-03,
+                  "optimization.end_lr": 3e-03,
+                  "optimization.max_epochs": 3,
+                  "optimization.warmup_steps": 0,
+                  "env.batch_size": 32,
+              })
+```
+
+The fraction of the tunable parameters is around **0.5%** of all parameters. Actually, the model can still achieve good performance on the test sets (even in the cross-lingual transfer setting). It obtained **comparable results** as full-finetuning as in :ref:`sec_automm_textprediction_multilingual`.
+
+
+```{.python .input}
+score_in_en = predictor.evaluate(test_en_df)
+score_in_de = predictor.evaluate(test_de_df)
+score_in_jp = predictor.evaluate(test_jp_df)
+print('Score in the English Testset:', score_in_en)
+print('Score in the German Testset:', score_in_de)
+print('Score in the Japanese Testset:', score_in_jp)
+```
+
+## Combine Gradient Checkpointing and Parameter-efficient Finetuning
+
+By combining [gradient checkpointing](https://pytorch.org/docs/stable/checkpoint.html) and parameter-efficient finetuning, it is feasible to finetune 
+models that have two billion parameters (e.g., [google/mt5-xl](https://huggingface.co/google/mt5-xl)) with a single GPU in [AWS G4 instances](https://aws.amazon.com/ec2/instance-types/g4/). 
+To turn on gradient checkpointing, you can just set up `"model.hf_text.gradient_checkpointing"` to `True`.
+
+```{.python .input}
+from autogluon.multimodal import MultiModalPredictor
+
+predictor = MultiModalPredictor(label="label",
+                                path="multilingual_ia3_gradient_checkpoint")
+predictor.fit(train_en_df,
+              presets="multilingual",
```

Comment:
The model used here is still `deberta` instead of the `t5` mentioned in the above description.

Reply:
Revised the text. I tried mT5-XL but the CI failed to build the tutorial. May be we can try it again when we are adding few-shot learning example.